trunk/SUAVE/Methods/Power/Battery/Sizing/initialize_from_circuit_configuration.py

```diff
@@ -50,8 +50,8 @@ def initialize_from_circuit_configuration(battery,module_weight_factor = 1.42):
     nominal_voltage              = battery.cell.nominal_voltage       
     total_battery_assemply_mass  = battery.cell.mass * battery.pack_config.series * battery.pack_config.parallel   
     battery.mass_properties.mass = total_battery_assemply_mass*module_weight_factor  
-    battery.specific_energy      = (amp_hour_rating*nominal_voltage)/battery.cell.mass  * Units.Wh/Units.kg   
-    battery.max_energy           = total_battery_assemply_mass*battery.specific_energy    
+    battery.specific_energy      = (amp_hour_rating*nominal_voltage)/battery.cell.mass   
+    battery.max_energy           = total_battery_assemply_mass*battery.specific_energy*(Units.Wh/Units.kg)      
```

Comment:
I think this should be returned to the way it was; battery specific energy is typically J/kg, so I think the change you made here mixes up the units

Reply:
Changed back